modules/lvms-deleting-lvmcluster-using-rhacm.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-deleting-lvmcluster-using-rhacm_{context}"]
+= Deleting an LVMCluster CR by using {rh-rhacm}
+
+If you have installed {lvms-first} by using {rh-rhacm-first}, you can delete an `LVMCluster` CR by using {rh-rhacm}.
+
+.Prerequisites
+
+* You have access to the {rh-rhacm} cluster as a user with `cluster-admin` permissions.
+* You have deleted the persistent volume claims (PVCs), volume snapshots, and volume clones provisioned by {lvms}. You have also deleted the applications that are using these resources.
+
+.Procedure
+
+. Log in to the {rh-rhacm} CLI using your {product-title} credentials.
+. Delete the `ConfigurationPolicy` CR YAML file that was created for the `LVMCluster` CR:
```

Comment:
```suggestion
. Delete the `ConfigurationPolicy` CR that was created for the `LVMCluster` CR:
```
Here we delete the CR from the cluster not the file.

Reply:
Updated as per the suggestion.